features/contract-state/GeneralInfo.tsx

```diff
@@ -124,14 +124,18 @@ const GeneralInfo = () => {
         </Status>
 
         <Status>
-          <Label>Is Expired: </Label>
-          <Link
-            href={DOCS_MAP.EXPIRY_SETTLEMENT}
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            <strong>{isExpired}</strong>
-          </Link>
+          <Label>
```

Comment:
change made to be consistent. looks like this:
![image](https://user-images.githubusercontent.com/12886084/89039366-3a70e180-d342-11ea-9b1f-d919945d9248.png)